cmake/package_nsis.cmake

```diff
@@ -34,22 +34,25 @@
 
 ## Windows installer
 
-find_program(MAKENSIS_EXE makensis)
-
-if (NOT MAKENSIS_EXE)
-  MESSAGE(FATAL_ERROR "Could not find 'makensis.exe'. Please make sure it's in $PATH!")
-endif()
-
-## check for correct NSIS version
-execute_process(COMMAND ${MAKENSIS_EXE} /HDRINFO
-                OUTPUT_VARIABLE NSIS_INFO 
-                COMMAND_ERROR_IS_FATAL ANY)                
-STRING(FIND ${NSIS_INFO} "Size of each section is 16408 bytes" NSIS_IS_8K) ## the 1k version gives "2072 bytes"
-
-if (NSIS_IS_8K EQUAL -1)
-  MESSAGE(FATAL_ERROR "NSIS (makensis.exe) needs to be the 'special build', which allows for 8k-length strings. This seems to be the 1k version. Please update NSIS. See https://github.com/OpenMS/NSIS")
-else()
-  MESSAGE(STATUS "Found 8k version of NSIS. Great!")
+# Check if nsis is actually run or if only the configuration is used
+if("${CPACK_GENERATOR}" STREQUAL "NSIS")
```

Comment:
Yeah, you could avoid that if everything in package_nsis was really just related to the package type, not the OS. But you can check what is needed first.